README.md

```diff
@@ -244,6 +244,72 @@ RAILS_ENV=production bundle exec rake drop[fixture_sr1]
 RAILS_ENV=production bundle exec rake populate[fixture_sr1]
 ```
 
+## <a name="console"/>Audit Checks via Console
+To run audit checks via the console open up the rails console:
+```sh
+bundle exec rails console production
+```
+
+Next, require the audit module. This will load all of the required files needed to run the audit checks.
+```ruby
+require 'audit'
+```
+
+Here are the list of the Audit Checks.
+
+### Moab To Catalog Audit Checks:
+
+#### Single Druid
+```ruby
+MoabToCatalog.check_existence_for_druid('xx000xx0000)
+```
+#### List of Druids
+```ruby
+MoabToCatalog.check_existence_for_druid_list('/path/to/your/csv/druid_list.csv')
+```
+#### Single Storage Root
+```ruby
+MoabToCatalog.check_existence_for_dir('path/to/root/spec/fixtures/storage_root01/moab_storage_trunk')
+```
+#### All Storage Roots
+```ruby
+MoabToCatalog.check_existence_for_all_storage_roots
+```
+
+### Catalog To Moab Audit Checks:
+-  The (date/timestamp) argument is a threshold:  it will run the check on all catalog entries which last had a version check BEFORE the argument. It should be in the format '2018-01-22 22:54:48 UTC'.
+
+- Note: Must enter date/timestamp argument as a string.
+#### Single Storage Root
+
+```ruby
+CatalogToMoab.check_version_on_dir('2018-01-22 22:54:48 UTC', 'path/to/root/spec/fixtures/storage_root01/moab_storage_trunk')
+```
+#### All Storage Roots
+```ruby
+CatalogToMoab.check_version_all_dirs('2018-01-22 22:54:48 UTC')
+```
+
+### Checksum Audit Checks:
+
+#### Single Storage Root
+```ruby
+Checksum.validate_disk_profiled('fixture_sr3')
```

Comment:
i would be inclined to suggest the non-profiled version of the method.

Reply:
copy pasta, thanks!